### PR TITLE
Homebank 5.0.0

### PR DESCRIPTION
**Pull request by AAS**
New major version of Homebank.

### CODE REVIEW

File: Library/Formula/homebank.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def install
     args << "--with-ofx" if build.with? 'libofx'
 
     system "./configure", *args
-    system "chmod +x ./install-sh"
+    chmod 0755, install-sh
```
Comment (asparagui) [inline Library/Formula/homebank.rb]: Does +x not do what you need?

Comment (AAS) [inline Library/Formula/homebank.rb]: ```brew audit``` fails with ```Use the `chmod` Ruby method instead of `system "chmod ````  output

**Comment by asparagui** [comment]
@BrewTestBot test this please

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**